lib/convection/model/template/resource/aws_s3_bucket.rb

```diff
@@ -50,6 +50,50 @@ def website_configuration(&block)
             properties['WebsiteConfiguration'].set(config)
           end
 
+          def full_name(terraform_import: false)
+            return "#{name.downcase}.#{stack._original_cloud}.#{stack._original_region}.rapid7.com" if terraform_import
+
+            "#{name.downcase}.#{stack.cloud}.#{stack.region}.rapid7.com"
+          end
+
+          def hcl_imports(module_path: 'root')
```

Comment:
1. `hcl_imports` should be `terraform_import_commands` otherwise it will not be called automatically when `terraform-export` is run.